docker/worker/Dockerfile_x86

```diff
@@ -0,0 +1,31 @@
+#!BuildTag: openqa_worker_x86
+FROM opensuse/leap:15.1
+LABEL maintainer Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz <wnereiz@eienteiland.org>, Sergio Lindo Mansilla <slindomansilla@suse.com>
+LABEL version="0.3"
+
+RUN zypper ar -f -G http://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.1/openSUSE_Leap_15.1 openQA
```

Comment:
Is there a ticket explaining why `http://` must be used? Is this x86_64-specific?